sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1377,6 +1377,15 @@ static bool _resolve_addr(const ipv6_addr_t *dst, gnrc_netif_t *netif,
     DEBUG("nib: resolve address %s by probing neighbors\n",
           ipv6_addr_to_str(addr_str, dst, sizeof(addr_str)));
     if (entry == NULL) {
+        /* don't do multicast address resolution on 6lo */
+        if (gnrc_netif_is_6ln(netif)) {
```

Comment:
You can move this outside the `if (entry == NULL)` - we don't do multicast address resolution in 6lo, so if we don't have a neighbor cache entry, we can exit early.